### PR TITLE
Added the Tint Color, Factor and Stroke Thickness attributes to GP Layer object

### PR DESCRIPTION
**Pull request by 3DSinghVFX**
I have added the _Tint Color, Factor_ and _Stroke Thickness_ attributes to GP Layer object. These attributes allow adding variation in duplicated layers (multiple layers). For example, Tint Color (based on the Factor) allows adding variation in the color for multiple layers which have only one material. The _Stroke Thickness_ allows changing the thickness of the strokes of a layer.
Screenshot from 2020-02-14 01-51-14

Examples,
Screenshot from 2020-02-14 01-48-59

Screenshot from 2020-02-14 02-27-26

### CODE REVIEW

**Comment by OmarEmaraDev** [comment]
*Factor* should be called *Tint Factor*.

**Comment by 3DSinghVFX** [comment]
> _Factor_ should be called _Tint Factor_.

Renamed.

File: animation_nodes/nodes/gpencil/gp_layer_from_frames.py [hunk 1]
```diff
@@ -16,12 +16,15 @@ def create(self):
             ("Frame", "frame"), ("Frames", "frames")), dataIsModified = True)
         self.newInput("Text", "Name", "layerName", value = 'AN-Layer')
         self.newInput("Text", "Blend Mode", "blendMode", value = 'REGULAR', hide = True)
-        self.newInput("Float", "Opacity", "opacity", value = 1, hide = True)
-        self.newInput("Integer", "Pass Index", "passIndex", value = 0, hide = True)
+        self.newInput("Float", "Opacity", "opacity", value = 1, minValue = 0, maxValue = 1, hide = True)
+        self.newInput("Color", "Tint Color", "tintColor", hide = True)
+        self.newInput("Float", "Factor", "tintFactor", value = 0, minValue = 0, maxValue = 1, hide = True)
```
Comment (OmarEmaraDev) [inline animation_nodes/nodes/gpencil/gp_layer_from_frames.py]: Tint Factor.

Comment (3DSinghVFX) [reply animation_nodes/nodes/gpencil/gp_layer_from_frames.py]: Renamed.

**Pull request merged**